primitives/src/ztg.rs

```diff
@@ -0,0 +1,39 @@
+// Distribution
+
+/// Total ZTG amount for airdrops
+pub const AIRDROPS: u128 = 3_000_000;
+
+/// Total ZTG amount for collators
+pub const COLLATORS: u128 = 10_000_000;
```

Comment:
This was invented with a random number and needs to be modified. It is also necessary to specify an amount of ZTG for liquidity mining.